src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        final int number = 100;
```

Comment:
this is not a constant field because you declare it in a method
please move it to the class level, and remember that constants should be static, final, and have PROPER_NAMING pattern